alpine/Dockerfile

```diff
@@ -46,8 +46,10 @@ RUN \
         autoconf \
         git \
     \
-    && if [ "${BUILD_ARCH}" = "armv7" ]; then \
+    && if [ "${BUILD_ARCH}" = "armv7" || "${BUILD_ARCH}" = "arm/v7" ]; then \
```

Comment:
`arm/v7` is not an Home Assistant build architecture.

Reply:
How about armv7l ?

Reply:
On my QNAP it show arch armv7l and not amrv7

Reply:
Please could you check this ? https://github.com/home-assistant/core/issues/86589

Reply:
`armv7l` is not a valid build architecture in Home Assistant.
You are mixing up what this variable means, is related/provided by our builders, which can only provide `armhf`, `aarch64`, `armv7`, `amd64` and `i386`.

> Please could you check this ? https://github.com/home-assistant/core/issues/86589

There is no need to spam your issue everywhere. Thanks 👍 

../Frenck